### PR TITLE
fixes issue #8. 

### PR DESCRIPTION
**Pull request by ryanramage**
handle an error instance in the source array by destoying the stream and passing on that error

### CODE REVIEW

**Comment by ryanramage** [comment]
@mafintosh if merged could you version bump and release. thanks!

File: test.js [hunk 1]
```diff
@@ -84,3 +84,38 @@ test('obj arrays', function (t) {
...
   })
 })
+
+
+test('arrays can emit errors', function (t) {
+  var input = ['a', 'b', new Error('ooops'), 'c']
+  var stream = from(input)
+  var output = []
+  stream.on('data', function (letter) {
+    output.push(letter.toString())
```
Comment (mafintosh) [inline test.js]: the output array is not used?

Comment (ryanramage) [inline test.js]: Sure l I could check that 2 elements have been added to the array before the error is introduced...

**Comment by mafintosh** [comment]
:+1: as soon as travis is done i'll merge it

**Comment by mafintosh** [comment]
Great! Released as 2.1.0

**Pull request merged**